lib/src/accordion/Accordion.tsx

```diff
@@ -1,15 +1,23 @@
-// @ts-nocheck
-import React, { useState } from "react";
+import React, { useId, useState } from "react";
 import styled, { ThemeProvider } from "styled-components";
-import ExpansionPanel from "@material-ui/core/ExpansionPanel";
-import ExpansionPanelSummary from "@material-ui/core/ExpansionPanelSummary";
-import ExpansionPanelDetails from "@material-ui/core/ExpansionPanelDetails";
-import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
 import { getMargin } from "../common/utils.js";
 import { spaces } from "../common/variables.js";
 import useTheme from "../useTheme";
 import { BackgroundColorProvider } from "../BackgroundColorContext";
-import AccordionPropsType from "./types";
+import AccordionPropsType, { Margin, Padding, Space } from "./types";
+import DxcFlex from "../flex/Flex";
+
+const expand_less = (
```

Comment:
What do you think about changing this variable name to camel case?

Reply:
Agree! I don't know why I used "_" to be fair.